Apps/pholetools/oletools.json

```diff
@@ -0,0 +1,167 @@
+{
+    "appid": "6549eb4a-d15c-4e01-8e15-91b85dd77087",
+    "name": "oletools",
+    "description": "This app provides actions to analyze Microsoft OLE2 files such as Microsoft Office Documents of Outlook messages. It is based on the oletools toolkit",
+    "type": "information",
+    "product_vendor": "decalage2",
+    "logo": "logo_splunk.svg",
+    "logo_dark": "logo_splunk_dark.svg",
+    "product_name": "oletools",
+    "python_version": "3",
+    "product_version_regex": ".*",
+    "publisher": "Splunk Community",
+    "license": "Copyright (c) 2021 Splunk Inc.",
+    "app_version": "1.0.1",
+    "utctime_updated": "2021-07-01T09:01:24.184114Z",
+    "package_name": "phantom_oletools",
+    "main_module": "oletools_connector.py",
+    "min_phantom_version": "4.9.39220",
+    "app_wizard_version": "1.0.0",
+    "pip_dependencies": {
+        "wheel": [
+            {
+                "module": "colorclass",
+                "input_file": "wheels/colorclass-2.2.0.zip"
+            },
+            {
+                "module": "olefile",
+                "input_file": "wheels/olefile-0.46.zip"
+            },
+            {
+                "module": "oletools",
+                "input_file": "wheels/oletools-0.60-py2.py3-none-any.whl"
+            }
+        ]
+    },
+    "configuration": {},
+    "actions": [
+        {
+            "action": "test connectivity",
+            "identifier": "test_connectivity",
+            "description": "Validate the asset configuration for connectivity using supplied configuration",
+            "type": "test",
+            "read_only": true,
+            "parameters": {},
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "mraptor scan",
```

Comment:
Possibly make this `get mraptor scan` to be similar to `get scan` action

Reply:
or should it be `run mraptor scan` going off of the description below?

Reply:
let's change it with `run mraptor scan`.

Reply:
Changed action name to `run mraptor scan`